hnclient-v2/pom.xml

```diff
@@ -191,6 +191,8 @@
 							<exclude>META-INF/*.RSA</exclude>
 							<exclude>**/application.properties</exclude>
 							<exclude>**/log4j2.properties</exclude>
+							<exclude>**/configure-hni.bat</exclude>
+							<exclude>**/HNS-Client.exe</exclude>
```

Comment:
src/main/resources should really be reserved for artifacts that get deployed with the application. Batch scripts, executables would be better suited to a /bin directory. This directory should not be picked up by maven.

Reply:
added a new folder hns-client-installer. Moved these files to that location. We can use the same folder in automation job to move the required contents from src/ and create a zip